cfgmgr/teammgr.cpp

```diff
@@ -400,12 +403,18 @@ bool TeamMgr::addLag(const string &alias, int min_links, bool fallback)
         << " -t " << alias
         << " -c " << conf.str()
         << " -L " << dump_path
-        << " -d";
-    EXEC_WITH_ERROR_THROW(cmd.str(), res);
+        << " -g -d";
```

Comment:
Does -g give give proper log Verbosity Level?

Reply:
yes the debug logs will be in teamd.log file under `/var/log` folder